src/SDK/Resource/Detectors/Composer.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Perhaps we could come up with another naming convention here?  When I read `Composer`, I immediately went to [The PHP dependency manager.](https://getcomposer.org/)